pymc/distributions/continuous.py

```diff
@@ -2518,14 +2517,38 @@ def logcdf(value, nu):
 
 
 # TODO: Remove this once logp for multiplication is working!
-class WeibullBetaRV(WeibullRV):
-    ndims_params = [0, 0]
+class WeibullBetaRV(RandomVariable):
+    name = "weibull"
+    ndim_supp = 0
+    ndims_params = [0]
+    dtype = "floatX"
+    _print_name = ("Weibull", "\\operatorname{Weibull}")
+
+    def __call__(self, alpha, beta, size=None, **kwargs):
+        return super().__call__(alpha, beta, size=size, **kwargs)
 
     @classmethod
     def rng_fn(cls, rng, alpha, beta, size) -> np.ndarray:
         return np.asarray(beta * rng.weibull(alpha, size=size))
 
 
+@_logprob.register(WeibullBetaRV)
```

Comment:
For the time being we should keep it consistent and implement it as the fake `logp` method of the PyMC Weibull distribution class, as is done for other distributions in this module.

In that case you don't need to do the value unwrap thing either.

Reply:
Ok, I will try like you say.